core/src/epicli/data/common/validation/configuration/recovery.yml

```diff
@@ -17,3 +17,75 @@ properties:
     properties:
       components:
         type: object
+        additionalProperties: false
+        properties:
+          load_balancer:
+            "$id": "#/properties/specification/properties/components/properties/load_balancer"
+            type: object
+            required:
+            - enabled
+            - snapshot_name
+            additionalProperties: false
+            properties:
+              enabled:
+                "$id": "#/properties/specification/properties/components/properties/load_balancer/properties/enabled"
+                type: boolean
+              snapshot_name:
+                "$id": "#/properties/specification/properties/components/properties/load_balancer/properties/snapshot_name"
+                type: string
```

Comment:
Would be amazing if we'd get validation for it (and all the others)! It can be either `latest` or `20200526-102034` (just digits and single '-'). :+1: What do you think?

Reply:
Great idea. It's done now in 90e478f.